src/Properties/Locale.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace GingerPluginSdk\Properties;
+
+use GingerPluginSdk\Bases\BaseField;
+use GingerPluginSdk\Helpers\FieldsValidatorTrait;
+use GingerPluginSdk\Interfaces\ValidateFieldsInterface;
+
+final class Locale extends BaseField implements ValidateFieldsInterface
+{
+    use FieldsValidatorTrait;
+
+    /**
+     * @throws \GingerPluginSdk\Exceptions\OutOfPatternException
+     */
+    public function validate($value)
+    {
+        $this->validatePattern(
+            value: $value,
+            pattern: "/^[a-zA-Z]{2}([\\\-_][a-zA-Z]{2})?$/",
+            /** @TOOD: Update examples of valid values. */
+            valid: ['UA_ua,NL_nl,NL_be,FR_de']
```

Comment:
I would not validate actual locale's since they change separately from your library. The pattern validation seems fair to me.

Reply:
For this moment i think i duplicate my answers on such questions :) Thought you could just miss some minor parts, from Exception `OutOfPattern`. 

> The pattern validation seems fair to me.

Ofk, i used a direct pattern from order.json schema of the Ginger API :D

Reply:
It would be great if you can keep the pattern but remove the valid options. The valid options don't change out of sync with this library. Keeping it in requires constant maintenance. Also I don't think we have any limitation here. If it's not supported we default to English.

Reply:
Logical, will implement!